AutomationApp/TokenHandler.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using System.Runtime.Serialization.Json;
+using Microsoft.Identity.Client;
+
+namespace AutomationApp
+{
+    public class TokenHandler
+    {
+        #region Properties
+        private static readonly Dictionary<string, string> JsonLabelReplacements = new Dictionary<string, string>();
+        public User CurrentUser { get; set; }
+
+        private PublicClientApplication _publicClientApplication;
+       #endregion
+
+        #region Constructor
+        static TokenHandler()
+        {
+            JsonLabelReplacements.Add("AccessTokenType", "access_token_type");
+            JsonLabelReplacements.Add("AccessToken", "access_token");
+            JsonLabelReplacements.Add("ExpiresOn", "expires_on");
+            JsonLabelReplacements.Add("ExtendedExpiresOn", "extended_expires_on");
+            JsonLabelReplacements.Add("ExtendedLifeTimeToken", "extended_lifetime_token");
+            JsonLabelReplacements.Add("IdToken", "id_token");
+            JsonLabelReplacements.Add("TenantId", "tenant_id");
+            JsonLabelReplacements.Add("UserInfo", "user_info");
```

Comment:
there is no user_info returned. please check what values are being returned and update this list accordingly.